util/topgen.py

```diff
@@ -815,6 +815,8 @@ def generate_top_ral(top, ip_objs, out_path):
             mem.name = item["name"]
             mem.base_addr = int(item["base_addr"], 0)
             mem.limit_addr = int(item["base_addr"], 0) + int(item["size"], 0)
+            mem.byte_write = ('byte_write' in item and
+                              item["byte_write"].lower() == "true")
```

Comment:
I wonder whether it would be better to "parse" the text as a bool and then check whether it's true? I think that I'd prefer to see an error if I added `byte_write: cabbage` in my hjson.

Reply:
it's pretty easy to add a check for this, but there are plenty of these attributes using similar text (like true, false, yes, no etc) in topgen and tlgen. Adding this check one by one seems not a good idea as it makes codes complex. Or maybe we can rely on DV flow or document review to find out this type of error. WDYT?

Reply:
Hmm, ok. The "right" thing to do is probably to write proper parsers that turn the hjson dicts into objects with proper schemas. Obviously, that's miles out of scope of this PR!

Reply:
agree, that is not the propose of this PR